src/Controls/src/Core/Platform/Android/PointerGestureHandler.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using Android.Views;
+using Microsoft.Maui.Controls.Platform.Android;
+using Microsoft.Maui.Graphics;
+using AView = Android.Views.View;
+
+namespace Microsoft.Maui.Controls.Platform
+{
+	internal class PointerGestureHandler
+	{
+		internal PointerGestureHandler(Func<View> getView, Func<AView> getControl)
+		{
+			GetView = getView;
+			GetControl = getControl;
+			SetupHandlerForPointer();
+		}
+
+		Func<View> GetView { get; }
+		Func<AView> GetControl { get; }
+
+		public void SetupHandlerForPointer()
+		{
+			var view = GetView();
+			if (view == null)
+				return;
+
+			var control = GetControl();
+			if (control == null)
+				return;
+
+			foreach (var gesture in view.GetCompositeGestureRecognizers())
+				if (gesture is PointerGestureRecognizer)
+					OnPointerAction(view, control, gesture as PointerGestureRecognizer);
+			
+			return;
+		}
+
+		private void OnPointerAction(View view, AView control, PointerGestureRecognizer pgr)
+		{
+			if (pgr == null)
+				return;
+
+			control?.SetOnHoverListener(CreatePointerRecognizer((v, e) =>
```

Comment:
Should the `OnHoverListener` loop over the `CompositeGestureRecognizers` and then call these methods on any found `PointerGestureRecognizers`? Users might want to add multiple `PointerGestureRecognizers`

Reply:
hmm... so I think that's what I was trying to achieve here: https://github.com/dotnet/maui/pull/10352#discussion_r988392078

but maybe I'm misinterpreting? we can chat more offline

Reply:
ok I think I got it now!